src/main/java/frc/robot/subsystems/endEffector/EndEffectorIO.java

```diff
@@ -1,12 +1,12 @@
 package frc.robot.subsystems.endEffector;
 
-interface EndEffectorIO {
+public interface EndEffectorIO {
```

Comment:
Tbh this feels extremely hacky of a fix. I'm wondering. I'm wondering whether setting this interface to `protected` would ensure integrity